docs/vsphere_ci/README.md

```diff
@@ -16,169 +21,113 @@ In case of our vSphere environment, we need to install Packer 1.6.6 on the basti
+
 
-## Prerequsite files
-- Please ensure `scripts` directory is present in the location where you are running Packer from on the bastion and 
-has the following files:
+## Prerequisite files
 
-- autounattend.xml
-- install-vm-tools.cmd
-- install-openssh.ps1
-- install-firewall-rules.ps1 
-- authorized_keys
-- install-docker.ps1
+Please ensure the `scripts` directory is present in the location where you are running Packer from on the 
+bastion host and has the following files:
 
-The `scripts/authorized_keys` file must be edited to contain a public key. The private key associated with the public 
-key is what will be used by WMCO to configure VMs created from Windows VM. After deploying WMCO, this private key will 
-be provided by the user in the form of a Secret.
+    - authorized_keys
+    - autounattend.xml
+    - install-vm-tools.cmd
+    - install-openssh.ps1
+    - install-docker.ps1
+    - install-firewall-rules.ps1
 
-The `scripts/autounattend.xml` file must be edited to change the value of `WindowsPassword` to a user provided password.
+The [authorized_keys](scripts/authorized_keys) file must contain a public key, where the private key 
+associated with this public key is what will be used by WMCO to configure VMs created from Windows VM. After 
+deploying WMCO, this private key will be provided by the user in the form of a Secret.
 
-The provided [autounattend.xml](scripts/autounattend.xml)
-- Installs VMWare tools
+The [autounattend.xml](scripts/autounattend.xml) file must be edited to update the value of 
+`_ADMINISTRATOR_PASSWORD_` with a user provided password. Then, it executes the following steps:
+
+- Runs `install-vm-tools.cmd` script which installs VMWare tools
 - Runs `install-openssh.ps1` script which installs and configures OpenSSH Server
-- Runs `install-firewall-rules.ps1` script which configures the firewall rules
 - Runs `install-docker.ps1` script which installs Docker
+- Runs `install-firewall-rules.ps1` script which configures the firewall rules
 
-This autounattend script is different from the [autounattend script](../unattend.xml) as this script does Windows OS
-installation as well.
+The above [autounattend.xml](scripts/autounattend.xml) script is different from the [unattend.xml](../unattend.xml)
+as this script does Windows OS installation as well.
 
-References:
-- [Sample autounattend](https://github.com/guillermo-musumeci/packer-vsphere-iso-windows/blob/master/win2019.base/win2019.base.json)
-- [Packer unattended windows installs](https://www.packer.io/guides/automatic-operating-system-installs/autounattend_windows)
 
-### Sample Packer build file
+## Packer build configuration file
+
+Packer needs a build file which specifies how the virtual machine template should be built. You can find a [reference 
+build file](build.json) in the repository.
 
-Packer needs a build file which specifies the how the VM template should be built. A sample build file is shown below.
-```
-{
```

Comment:
Removed this block to avoid duplicated definitions, since the build.json file is included.